Automattic/simplenote-ios
# Simplenote Sustainer UI

### Fix
In this PR we're implementing the Simplenote Sustainer UI.

@frosty Can I bug you with this one?
Thank yooouuuu!!

Ref. #1491

### Test: iOS +15
1. Launch Simplenote iOS on a device running iOS 15 or 16
2. Open the Sidebar

- [x] Verify the **Become  a Subscriber** UI shows up at the top of the table
- [x] Verify that pressing over such Header causes the UI to toggle between `Become a Subscriber` / `Active Subscriber` modes
- [x] Verify this UI looks great in Light / Dark Mode


### Test: iOS 14
1. Launch Simplenote iOS on a device running iOS 14
2. Open the Sidebar

- [x] Verify the Subscriber UI doesn't show up
- [x] Verify the Subscriber UI isn't visible, either, at the top of the Settings UI

### Release
These changes do not require release notes.

### Screenshots

Become Subscriber | Active Subscriber
-|-
<img width="300" src="https://user-images.githubusercontent.com/1195260/198293883-4ffc7c51-5051-4374-8255-bbc3f3f0eee9.png"> | <img width="300" src="https://user-images.githubusercontent.com/1195260/198293895-e878733b-f1c1-4d62-9767-3b48c3cdf53d.png">

Become Subscriber | Active Subscriber
-|-
<img width="300" src="https://user-images.githubusercontent.com/1195260/198294314-aef5afaa-8d03-4f39-8847-5ffa7ea954b2.png"> | <img width="300" src="https://user-images.githubusercontent.com/1195260/198294331-e7691ac8-c2f5-4d08-a253-07da11bb0e9d.png">
